behind-the-veil-siteroot/server/main.js

```diff
@@ -30,11 +30,18 @@ import "/imports/api/collections/messages.js";
 import "/imports/api/methods/messages.js";
 import "/imports/api/publications/messages.js";
 
+
 import "/server/mailer.js"
 import {checkBookingsEveryMidnight} from "./background.js"
 
 // file containing creds for mail server
 import {mailUrl, fromUser} from "./secrets.js"
+import "/imports/api/collections/reviews.js";
+import "/imports/api/methods/reviews.js";
+import "/imports/api/publications/reviews.js";
```

Comment:
Probably want to write some test cases to see if this works

Reply:
i think it has been already been created n the test folder

Reply:
Only for getting and adding a review, not for ur new method i think